### PR TITLE
Added full UTF-8 support in Labels

### PR DESCRIPTION
**Pull request by andreborud**
Added it in my private project and thought other might like it as well. 
I have added a third-party library called [GraphemeSplitter](https://github.com/orling/grapheme-splitter) which I have slightly modified to be able to import it with define().
The other change is to LabelCollection where it now splits double characters like emojis correctly.

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thank you so much for the pull request @andreborud! I noticed this is your first pull request and I wanted to say welcome to the Cesium community!

The [Pull Request Guidelines](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#pull-request-guidelines) is a handy reference for making sure your PR gets accepted quickly, so make sure to skim that.

* :x: Missing `CONTRIBUTORS.md` entry.
    * Please add yourself to the [contributors](https://api.github.com/repos/AnalyticalGraphicsInc/cesium/contents/CONTRIBUTORS.md) file!
* :x: Missing CLA.
    * Please send in a [Contributor License Agreement](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#contributor-license-agreement-cla) (CLA) and comment back here to let us know to check this!
* :grey_question: Changes to third party files were made.
    * Looks like a file in one of our ThirdParty folders (`ThirdParty/, Source/ThirdParty/`) has been added or modified. Please verify that it has a section in [LICENSE.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/LICENSE.md) and that its license information is up to date with this new version.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.
- [ ] Works (or fails gracefully) in IE11.

---

__I am a bot who helps you make Cesium awesome! [Contributions to my configuration are welcome.](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/.concierge)__

:earth_africa: :earth_americas: :earth_asia:

**Comment by andreborud** [comment]
Thanks @pjcozzi. 

I'm realising "Added support for emojis in Labels" might not be the best description for what this pull request adds. We had two problems that got solved simultaneously one was that we needed to use national flags which we used billboards for which reduced the performance for us, but using emoji national flags in a label instead reduced the number of entities we had to draw. 

The other problem was that cesium didn't support many non-english characters which we needed. So maybe this pull request should be called "Added full UTF-8 support for labels". 

Can't say I'm certain this is the best approach and I haven't made any extensive performance/memory testing. 
Could it be a better idea to make this optional for labels or to only apply it if any of the characters in the label has a higher character number than 128 as an example?

**Comment by cesium-concierge** [comment]
Thanks again for your contribution @andreborud! 

No one has commented on this pull request in 30 days. Maintainers, can you review, merge or close to keep things tidy?

 _I'm going to re-bump this in 30 days. If you'd like me to stop, just comment with `@cesium-concierge stop`. If you want me to start again, just delete the comment._

---

__I am a bot who helps you make Cesium awesome! [Contributions to my configuration are welcome.](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/.concierge)__

:earth_africa: :earth_americas: :earth_asia:

**Comment by mramato** [comment]
Sorry for the delay here.  I'm interested in getting this reviewed and merged, since as you mentioned it's more about UTF-8 support) but I need to take a look at your library and better understand the issue all around.  I'll try to get to it before the next release.

**Comment by cesium-concierge** [comment]
Thanks again for your contribution @andreborud! 

No one has commented on this pull request in 30 days. Maintainers, can you review, merge or close to keep things tidy?

 _I'm going to re-bump this in 30 days. If you'd like me to stop, just comment with `@cesium-concierge stop`. If you want me to start again, just delete the comment._

**Comment by hpinkos** [comment]
@andreborud please merge in master when you have a chance.  @mramato will hopefully be able to look at this for the next release.  Thanks!

**Comment by cesium-concierge** [comment]
Thanks again for your contribution @andreborud! 

No one has commented on this pull request in 30 days. Maintainers, can you review, merge or close to keep things tidy?

 _I'm going to re-bump this in 30 days. If you'd like me to stop, just comment with `@cesium-concierge stop`. If you want me to start again, just delete the comment._

**Comment by cesium-concierge** [comment]
Thanks again for your contribution @andreborud! 

No one has commented on this pull request in 30 days. Maintainers, can you review, merge or close to keep things tidy?

 _I'm going to re-bump this in 30 days. If you'd like me to stop, just comment with `@cesium-concierge stop`. If you want me to start again, just delete the comment._

**Comment by cesium-concierge** [comment]
Thanks again for your contribution @andreborud! 

No one has commented on this pull request in 30 days. Maintainers, can you review, merge or close to keep things tidy?

 _I'm going to re-bump this in 30 days. If you'd like me to stop, just comment with `@cesium-concierge stop`. If you want me to start again, just delete the comment._

**Comment by cesium-concierge** [comment]
Thanks again for your contribution @andreborud! 

No one has commented on this pull request in 30 days. Maintainers, can you review, merge or close to keep things tidy?

 _I'm going to re-bump this in 30 days. If you'd like me to stop, just comment with `@cesium-concierge stop`. If you want me to start again, just delete the comment._

**Comment by cesium-concierge** [comment]
Thanks again for your contribution @andreborud! 

No one has commented on this pull request in 30 days. Maintainers, can you review, merge or close to keep things tidy?

 _I'm going to re-bump this in 30 days. If you'd like me to stop, just comment with `@cesium-concierge stop`. If you want me to start again, just delete the comment._

**Comment by cesium-concierge** [comment]
Thanks again for your contribution @andreborud! 

No one has commented on this pull request in 30 days. Maintainers, can you review, merge or close to keep things tidy?

 _I'm going to re-bump this in 30 days. If you'd like me to stop, just comment with `@cesium-concierge stop`. If you want me to start again, just delete the comment._

**Comment by cesium-concierge** [comment]
Thanks again for your contribution @andreborud! 

No one has commented on this pull request in 30 days. Maintainers, can you review, merge or close to keep things tidy?

 _I'm going to re-bump this in 30 days. If you'd like me to stop, just comment with `@cesium-concierge stop`. If you want me to start again, just delete the comment._

**Comment by cesium-concierge** [comment]
Thanks again for your contribution @andreborud! 

No one has commented on this pull request in 30 days. Maintainers, can you review, merge or close to keep things tidy?

 _I'm going to re-bump this in 30 days. If you'd like me to stop, just comment with `@cesium-concierge stop`. If you want me to start again, just delete the comment._

**Comment by andreborud** [comment]
Sorry for the delay. Master branch merged.

**Comment by mramato** [comment]
@andreborud I apologize again for it taking so long to get this merged. I think "support for emojis" is definitely underselling the improved support for non-latin character sets that this PR provides.  I'll tweak CHANGES in master, but otherwise this looks good to me.  Thanks!

I was slightly worried that this would affect label performance, but since it only happens at label generation time, it has negligible impact in the small amount of testing I did.  Plus if performance does turn out to be an issue, it would be trivial to make this opt-in.

File: CHANGES.md [hunk 1]
```diff
@@ -268,6 +268,7 @@ _This is an npm-only release to fix a publishing issue_.
 * Added `computeLineSegmentLineSegmentIntersection` to `Intersections2D`. [#7228](https://github.com/AnalyticalGraphicsInc/Cesium/pull/7228)
 * Added ability to load availability progressively from a quantized mesh extension instead of upfront. This will speed up load time and reduce memory usage. [#7196](https://github.com/AnalyticalGraphicsInc/cesium/pull/7196)
 * Added the ability to apply styles to 3D Tilesets that don't contain features. [#7255](https://github.com/AnalyticalGraphicsInc/Cesium/pull/7255)
+* Added support for emojis in Labels.
```
Comment (emackey) [inline CHANGES.md]: Looks like this got merged with "support for emojis", but more importantly, the CHANGES message is back-dated to Cesium 1.52 from last December.

I've often wondered if Cesium needs an automated process to detect and alert for changes to CHANGES for released versions.

Comment (emackey) [reply CHANGES.md]: Nevermind, Matt fixed it in e0e8062907d.

Comment (mramato) [reply CHANGES.md]: Sorry, meant to leave a comment on this PR linking to that.  Thanks @emackey!

**Pull request merged**